nixpkgs.nix

```diff
@@ -0,0 +1,4 @@
+import (fetchTarball {
```

Comment:
Please move this file under `dev` to avoid polluting the root given that it is not necessary that it is there.

Reply:
Done.